truewiki/storage/git.py

```diff
@@ -165,6 +168,9 @@ def file_rename(self, old_filename: str, new_filename: str) -> None:
 
         super().file_rename(old_filename, new_filename)
 
+    def get_file_nonce(self, filename: str) -> str:
+        return self._git.head.commit.hexsha if self._git else "none"
```

Comment:
I would say: either fallback to the `super` (which is the local one) or use None. But using `"none"` is a bit silly :D